docs/metrics-3.6.md

```diff
@@ -0,0 +1,81 @@
+# Metrics upgrade notes
+
+Metrics have changed between 3.5 and 3.6.
```

Comment:
```suggestion
Metrics have changed in 3.6.
```

"between 3.5 and 3.6" is implicit, and it also affects versions prior to 3.5 as well, so remove that.
We also conventionally only state the version number things changed.